content/en/blog/falco-plugin-k8saudit-eks.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Monitoring your EKS clusters audit logs"
+linktitle: "Monitoring your EKS clusters audit logs"
+date: 2022-11-04
+author: Thomas Labarussias
+slug: k8saudit-eks-plugin
+---
+
+At the beginning of this year, Falco introduced a game changing feature, [Falco Plugins](https://falco.org/blog/falco-announcing-plugins/). They allow Falco to monitor and trigger alerts for any kind of event. We have since created plugins for [Github](https://github.com/falcosecurity/plugins/tree/master/plugins/github), [AWS CloudTrail](https://github.com/falcosecurity/plugins/tree/master/plugins/cloudtrail), [Okta](https://github.com/falcosecurity/plugins/tree/master/plugins/okta), and have replaced the way Falco consumes Audit Logs generated by a K8s API server through a [dedicated plugin](https://github.com/falcosecurity/plugins/tree/master/plugins/k8saudit). With these plugins, Falco covers more and more aspects of your infrastructure and allows you to use a single syntax for rules.
+
+For months (years...), our adopters have asked us for a way to monitor the [Audit Logs of K8s](https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/). The previous implementation used an internal web server to receive the logs from the Kubernetes API. Even if it worked well for manually installed and managed clusters, this method didn't fit the clusters managed by the cloud providers, like EKS, AKS, or GKE. They capture the audit logs for their own usages and expose them in their own log aggregator.
+
+This situation is now solved thanks to the plugin framework and we're proud to announce the first release of the plugin for the EKS Audit logs !!!
```

Comment:
This situation is now solved thanks to the plugin framework and we're proud to announce the first release of the plugin for EKS Audit logs!!!